examples/in-source-test/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@vitest/example-in-source-test",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "test": "vitest",
+    "test:run": "vitest run"
+  },
+  "devDependencies": {
+    "typescript": "^5.5.4",
+    "vite": "latest",
```

Comment:
Let's remove vite from deps since we don't use anything directly from there

Reply:
Removed vite from deps.
[8530001](https://github.com/vitest-dev/vitest/pull/6259/commits/8530001c776dd533f7d2af474c34f61f3717495b)